queue/group_remote_mongo.go

```diff
@@ -47,6 +47,9 @@ type MongoDBQueueGroupOptions struct {
 	// to each queue, based on the queue ID passed to it.
 	WorkerPoolSize func(string) int
 
+	// RetryHandlerOpts configure how retryable jobs are handled.
+	RetryHandler amboy.RetryHandlerOptions
```

Comment:
Should this be called `RetryHandlerOpts`?

Reply:
It's already in an options struct so I think it would be redundant.